rook/client_deploy.yml

```diff
@@ -0,0 +1,29 @@
+steps:
+- id: copy ceph config
+  uses: docker://bitnami/kubectl
+  runs: ['./scripts/copy_ceph_config.sh']
+
+- id: build ceph-client image
+  uses: docker://docker:19.03.3
+  args: [
+    'build',
+    '-t',
+    'jcnitdgp25/ceph-client:latest',
+    './ceph-client'
+  ]
```

Comment:
can you expand a bit on why we need to build a client image? Is the client not available on the [base ceph image](https://hub.docker.com/r/ceph/ceph)? If not, can we extend those instead?

Reply:
Yeah used the ceph base image. That works. Thanks